+nosnoc/+model/Cls.m

```diff
@@ -0,0 +1,180 @@
+classdef Cls < nosnoc.model.Base
```

Comment:
I made some edits in the CLS PR in this file (mostly docu, explanation).

Btw, stupid question, if we use time_freezing, do all CLS variables get created at all? All the Lambda, Y_gap and so on, or does it get removed (we dont need it for the later PSS).

Reply:
No. the FESD-J vars get created in the `dcs` step of the pipeline. Time freezing occurs at the `model` stage taking `nosnoc.model.Cls` and replacing it with `nosnoc.model.Pss` or `nosnoc.model.Heaviside` and feeding that into the rest of the pipeline.

Reply:
perfect!